src/overview.md

```diff
@@ -0,0 +1,126 @@
+---
+layout: page
+title: Overview
+---
+
+# Overview
+
+Workbox is a collection of libraries and build tools that make it easy to
+store your website's resources locally, on your users' devices. Consider Workbox
+if you want to:
+
+* Make your site work offline.
+* Improve load performance on repeat-visits. You can use Workbox to store
```

Comment:
Honestly, there isn't a performance benefit of using Workbox vs. using proper HTTP caching headers when it comes to loading subresources used a page. What Workbox lets you do that you couldn't do otherwise is fulfill the navigation request for the initial HTML document using the cache. Could this be rephrased to something like:

> Even when there is a network connection, respond with your site's HTML immediately, instead of having to wait for a response from your server.